sycl/test/usm/memadvise.cpp

```diff
@@ -0,0 +1,83 @@
+// RUN: %clangxx -fsycl %s -o %t1.out -lOpenCL
```

Comment:
AFAIK "-lOpenCL" is not needed as "-fsycl" adds it implicitly. If so, please remove.

Reply:
I don't think so.
I think, `-fsycl` enables linking of SYCL runtime only (on Linux), so I explicit linking with OpenCL is required.